### DISCUSSION TITLE
Alternating back to a "set of windows" (like StageManager does, but with StageManager disabled)

### DISCUSSION THREAD

**Discussion by ricfwolff** [rxhanson/Rectangle#1202] [Q&A]
I'm not sure this is related to Rectangle, but I'd be happy if someone points me to an app that does that:
**Sometimes** I have an application (eg. VSCode) on two-thirds and two other apps on the 2 sixths. Then I switch to my email app, or safari, or messaging. Then, when I want to go back to my previous set of windows (VSCode + 2 others), I have to alternate between all my open windows until I find those windows and focus back to the main one.
I use AltTab, if that's important.
PS: I have used StageManager in the past, but not in conjunction with Rectangle. I've disabled StageManager because sometimes I like having all the windows "available" and the situation I described above is not my default usage. So I don't think StageManager would be the answer to it.

**Comment by ricfwolff** [comment]
Maybe "Spaces" is the answer to my own question? 
I just remember that it existed and could be used to do so. I created a new Space with the apps and layout I want. And left email and messaging on the default space. I'll test it a bit today and tomorrow. :)

**Reply by rxhanson** [reply]
Right, I think Spaces is probably the most logical solution to what you're looking for. 

I do have a backlog item in Rectangle Pro for something that's sort of in between. Right now in Rectangle Pro, you can save a layout of windows, and restore that layout using a keyboard shortcut or a handful of other triggers. The backlog item I have is to hide all of the windows not included in the layout. That way you can use just one "Space", and have the benefit of multiple layouts. This is also due to Apple never making a publicly available API for Spaces, so I don't have a suitable way to work with Spaces as a 3rd party.

**Reply by ricfwolff** [reply]
Nice. That's a great feature. 😄
